cognite/neat/_session/_show.py

```diff
@@ -14,26 +18,147 @@ class ShowAPI:
     def __init__(self, state: SessionState) -> None:
         self._state = state
         self.data_model = ShowDataModelAPI(self._state)
+        self.instances = ShowInstanceAPI(self._state)
+
+
+class ShowInstanceAPI:
+    def __init__(self, state: SessionState) -> None:
+        self._state = state
+
+    def __call__(self) -> Any:
+        if not self._state.store.graph:
+            print("No graph data available.")
+            return None
+
+        widget = CytoscapeWidget()
+        widget.layout.height = "700px"
+
+        G, types = self._get_instance_di_graph_and_types()
+        widget_style = self._generate_cytoscape_widget_style(types)
+        widget.set_style(widget_style)
+
+        widget.graph.add_graph_from_networkx(G)
+        print("Max of 100 nodes and edges are displayed, which are randomly selected.")
+
+        return display(widget)
+
+    def _get_instance_di_graph_and_types(self):
```

Comment:
Should this be somewhere else? Or is it specific to the visualization. Seems generic to create a networkx graph from the triple store

Reply:
we will move it to `probe`